docs/cli_interface.md

```diff
@@ -187,6 +187,22 @@ explainaboard --task qa-extractive --dataset squad --system_outputs MY_FILE > re
 ```
 
 
+## [Open Domain QA](task_qa_open_domain.md)
```

Comment:
I didn't see `task_qa_open_domain.md` added, so is this link broken?
It also seems that the file format is not described?

Reply:
fixed in another pr: https://github.com/neulab/ExplainaBoard/pull/365